api/adaptive_load/metrics_plugin_impl.proto

```diff
@@ -0,0 +1,13 @@
+// Plugin-specific config protos for MetricsPlugin.
+
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+// Example plugin-specific config proto, representing connection info for an
+// outside source of metrics. Part of an example showing how to create and
+// register a MetricsPlugin.
+message ExampleMetricsPluginConfig {
```

Comment:
Additionally as per my other comment - I would suggest we keep ourselves focused on the minimal viable product and keep implementation of any advanced example into another chain of PRs.

Reply:
Agreed.